### ISSUE TITLE
"Last Success" doesn't seem to account for empty sources with no error.

### ISSUE THREAD

**Issue by ProtagNeptune** [Bionus/imgbrd-grabber#3251]
My Grabber Install with the post-filters `!("animated"|"animated_gif"|"animated_png")&(ext:avif|ext:gif|ext:jxl|ext:png|ext:webp)|(ext:bmp|ext:jpe|ext:jpeg|ext:jpg)`:
![image](https://github.com/user-attachments/assets/a6ad7876-08d2-46bd-a42c-10be26a859ac)

My Grabber Portable with the post-filters `("animated"|"animated_gif"|"animated_png")&(ext:avif|ext:gif|ext:jxl|ext:png|ext:webp)|(ext:mp4|ext:swf|ext:webm|ext:zip)`:
![image](https://github.com/user-attachments/assets/c503fa3f-1c92-4b6c-b22f-7bc5a047b6d9)

Small `monitors.json` excerpt:
```
{
	"monitors": [
		{
			"cumulated": 0,
			"delay": 86400,
			"download": true,
			"filenameOverride": "",
			"getBlacklisted": false,
			"interval": 604800,
			"lastCheck": "1970-01-01T00:00:00Z",
			"lastState": {
				"count": 0,
				"since": "",
				"state": ""
			},
			"lastSuccess": "",
			"notify": false,
			"pathOverride": "C:\\Users\\jelly\\Grabber\\#Monitors\\JackTheRipper",
			"postFilters": [""],
			"preciseCumulated": true,
			"query": {
				"tags": [
					"jack_the_ripper_(fate/apocrypha)"
				]
			},
			"sites": [
				"aibooru.online",
				"behoimi.org",
				"booru.allthefallen.moe",
				"danbooru.donmai.us",
				"gelbooru.com",
				"hypnohub.net",
				"konachan.com",
				"rule34.xxx",
				"tbib.org",
				"yande.re"
			]
		}
	],
	"version": 1
}
```

**Comment by Bionus** [comment]
Yes, no results are considered an error, as it's unexpected to have a monitor with no results at all. Why are you monitoring a search that doesn't bring any results? 😅 

The most common reason this happens is when a tag has been renamed for example, hence why it's not considered a success 🤔

**Comment by ProtagNeptune** [comment]
> In case it starts having some one day?
Yep.

**Comment by Bionus** [comment]
Then TBH it doesn't really matter if it's flagged as an error or not. The "last success" time is only used for skipping images older than it, after all.

Although I guess I could change the icon to a ⚠️ instead since it's not strictly an error either.

**Comment by ProtagNeptune** [comment]
I mean, it _successfully_ got images without error:
- No 403 Forbidden
- No Cloudflare firewall
- It grabbed available files
- It skips if there are no files
